coral-spark/src/main/java/com/linkedin/coral/spark/CoralSpark.java

```diff
@@ -42,23 +43,26 @@ public class CoralSpark {
 
   private final List<String> baseTables;
   private final List<SparkUDFInfo> sparkUDFInfoList;
-  private final String sparkSql;
   private final HiveMetastoreClient hiveMetastoreClient;
+  private final SqlNode sqlNode;
```

Comment:
I think `sparkSqlNode` is more accurate?

Reply:
Agreed, but let's introduce LanguageSqlNode terminology after the Coral IR APIs are completed and published.